modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -0,0 +1,389 @@
+////
+This module included in the following assemblies:
+-distr_tracing_otel/distr-tracing-otel-configuring.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-config-otel-collector_{context}"]
+= OpenTelemetry Collector configuration options
+
+:FeatureName: The {OTELName} Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+The OpenTelemetry Collector consists of three components that access telemetry data:
+
```

Comment:
These look like a candidate for [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#basic-description-list)

Reply:
Indeed. Fixed. Thank you.